constants.js

```diff
@@ -13,7 +13,10 @@
 // limitations under the License.
 
 const openEvent = 'opened';
-const reopenEvent = 'reopened';
+// Github action sends a different type of event.
+const openEventGithubActions = 'pull_request_target_opened';
```

Comment:
Why is `_target_` added here and below?

Reply:
We need `pull_request_target` to get access to the github secrets. More info [here](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target)

Reply:
Oh okay.